tests/admin_inlines/admin.py

```diff
@@ -23,6 +23,11 @@ class NonAutoPKBookTabularInline(admin.TabularInline):
     classes = ('collapse',)
 
 
+class NonAutoPKBookDescendantTabularInline(admin.TabularInline):
+    model = NonAutoPKBookDescendant
+    classes = ('collapse',)
```

Comment:
I guess `collapse` isn't needed, is it? Then you don't need the selenium test change.